### PR TITLE
Bump COS image to cos-stable-57-9202-64-0

### PR DESCRIPTION
**Pull request by fabioy**
Changing the default COS image to a current and stable m57 release.

### CODE REVIEW

File: cluster/gce/config-default.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ fi
 # variable. Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
 CVM_VERSION=${CVM_VERSION:-container-vm-v20170214}
-GCI_VERSION=${KUBE_GCI_VERSION:-gci-stable-56-9000-84-2}
+GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-57-9202-64-0}
```
Comment (Amey-D) [inline cluster/gce/config-default.sh]: Probably more location too - see https://github.com/kubernetes/kubernetes/pull/41819/files as a previous example of bumping up COS image version.

**Comment by fabioy** [comment]
@yujuhong I noticed that the "test/e2e_node/jenkins/non-cri_validation/benchmark-config.yaml" still point to a beta m56 COS image. Should I update them to the m57 stable one? I changed the ones under "cri_validation" because they were already at m56 stable.

**Comment by Amey-D** [comment]
LGTM

**Comment by dchen1107** [comment]
non-cri tests should be removed from main branch, but still in 1.6 branch. You might need to manually cherrypick that. 

/lgtm

**Review by dchen1107 — APPROVED** [review]

**Comment by dchen1107** [comment]
We already upgraded k8s on GCE to cos-beta-59-9460-20-0. No need this pr.

**Pull request closed without merging**